libtsuba/src/tsuba.cpp

```diff
@@ -224,14 +166,60 @@ tsuba::Stat(const std::string& rdg_name) {
     return rdg_res.error();
   }
 
-  RDGMeta meta = rdg_res.value();
+  RDGManifest meta = rdg_res.value();
   return RDGStat{
       .num_partitions = meta.num_hosts(),
       .policy_id = meta.policy_id(),
       .transpose = meta.transpose(),
   };
 }
 
+katana::Result<std::vector<tsuba::RDGView>>
+tsuba::ListAvailableViews(const std::string& rdg_dir) {
```

Comment:
I strongly think this should only return valid views for the latest version. Or take a version argument and return views for that version.